src/utils/constants/classNames.js

```diff
@@ -0,0 +1,10 @@
+export const CLASS_BUTTON_CREATE = "content__button--create";
+export const CLASS_TASK_INPUT = "task-input";
+export const CLASS_TASK_LIST = "task-list";
+export const CLASS_TASK_LIST_ITEM = "task-list__item";
+export const CLASS_SEARCH = "search";
+export const CLASS_NAVBAR = "navbar";
+export const CLASS_NAVBAR_LOGO = "navbar__logo";
+export const CLASS_NAVBAR_LOGO_IMG = "navbar__logo--img";
+export const CLASS_CONTENT = "content";
+export const CLASS_BUTTON = "button";
```

Comment:
Is there any convention to use the classes as const?

I prefer to use the class directly on the component as that will help to understand the design at the current context.